### PR TITLE
CC | runtime: Add configurable context timeout for StopVM

### PR DESCRIPTION
**Pull request by bpradipt**
Add configurable context timeout for StopVM similar to StartVM

Fixes: #6730

### CODE REVIEW

File: src/runtime/virtcontainers/remote.go [hunk 1]
```diff
@@ -134,6 +134,10 @@ func (rh *remoteHypervisor) AttestVM(ctx context.Context) error {
 
 func (rh *remoteHypervisor) StopVM(ctx context.Context, waitOnly bool) error {
 
+	// waitOnly doesn't make sense for remote hypervisor and suited for local hypervisor.
+	// Instead use a similar logic like StartVM to handle StopVM with timeout.
+	timeout := int(rh.config.RemoteHypervisorTimeout)
```
Comment (jepio) [inline src/runtime/virtcontainers/remote.go:139]: Does this need the same logic as `StartVM`:
```
	minTimeout := defaultMinTimeout
	if rh.config.RemoteHypervisorTimeout > 0 {
		minTimeout = int(rh.config.RemoteHypervisorTimeout)
	}
```

**Review by jepio — APPROVED** [review]

File: src/runtime/virtcontainers/remote.go (hunk 1 above)
Comment (bpradipt) [reply src/runtime/virtcontainers/remote.go:139]: I'll modify the code to use the defaultMinTimeout

Comment (fidencio) [reply src/runtime/virtcontainers/remote.go:139]: @bpradipt, if you're already modifying the code, please, also change "Ad" -> "Add" in the commit message.

**Review by stevenhorsman — APPROVED** [review]
LGTM, after the fix has gone in (and assuming the tests pass). Thanks Pradipta and Jeremi!

**Pull request merged**